src/Containers/SquareTable.tsx

```diff
@@ -86,6 +87,7 @@ export const SquareTable: React.FC<ISquareTable> = ({
     return (
         <div>
             {/** chairs top */}
+
```

Comment:
Why are there blank lines in the tree. The tree should be small anyways that blank lines are unessesary, lets remove these